src/brevitas/core/zero_point.py

```diff
@@ -65,13 +65,28 @@ def forward(self, zero_point: Tensor, scale: Tensor, bit_width: Tensor) -> Tenso
 
 class _ScaleShiftQuantZeroPoint(brevitas.jit.ScriptModule):
 
-    def __init__(self, zp_int_quant: Module) -> None:
+    def __init__(
+            self,
+            zp_int_quant: Module,
+            int_quant,
+            zero_point_shape: Tuple[int],
```

Comment:
Same as above for the type hint
```suggestion
            zero_point_shape: Tuple[int, ...],
```